### PR TITLE
Session persistence support

### PR DESCRIPTION
**Pull request by gmfeinberg**
This PR also updates dependency versions. Please review

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   ],
   "scripts": {
     "test": "mocha",
-    "docs": "jsdoc -c conf.json"
+    "docs": "/usr/local/bin/jsdoc --verbose -c conf.json ./node_modules/docdash"
```
Comment (y-polonsk) [inline package.json:20]: This is not going to work on Windows.  What is the reason for this change?  Can it be made platform-neutral?

File: package.json [hunk 2]
```diff
@@ -27,25 +27,24 @@
...
   "devDependencies": {
-    "eslint": "^6.8.0",
-    "eslint-plugin-require-path-exists": "^1.1.9",
-    "jsdoc": "^3.5.5",
-    "docdash": "^1.0.3",
-    "mocha": "^6.0.0",
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
-    "lodash": "^4.17.11",
+    "compare-versions": "^3.6.0",
```
Comment (y-polonsk) [inline package.json:32]: What are the changes here in the devDependencies section?

**Review by y-polonsk — COMMENTED** [review]
May I ask how the session cookies are used in this case?  What is the reason for this change?

File: package.json (hunk 1 above)
Comment (gmfeinberg) [reply package.json:20]: That was a mistake. I'd made that local change to get proper doc generated. The problem I ran into was that the jsdoc in ./node_modules did not properly generate the doc pages. If you (@y-polonsk ) can figure out why it doesn't work properly  that'd be great.

File: package.json (hunk 2 above)
Comment (gmfeinberg) [reply package.json:32]: They were generated. I'll remove this and other changes not related to the session code and we can look at them separately

File: package.json (hunk 1 above)
Comment (y-polonsk) [reply package.json:20]: "npm run docs" works fine for me.  What output do you get?
